README.md

```diff
@@ -22,31 +20,90 @@ ODK Aggregate can be deployed on Google's App Engine, enabling users to quickly
 * ODK developer Slack archive: [http://opendatakit.slackarchive.io](http://opendatakit.slackarchive.io) 
 * ODK developer wiki: [https://github.com/opendatakit/opendatakit/wiki](https://github.com/opendatakit/opendatakit/wiki)
 
-## Release cycle
+## Getting the code
 
-TBD
+1. Fork the Aggregate project ([why and how to fork](https://help.github.com/articles/fork-a-repo/))
 
-## Setting up your development environment
+1. Clone your fork of the project locally. At the command line:
 
-1. Download and install [Git](https://git-scm.com/downloads) and add it to your PATH
+        git clone https://github.com/YOUR-GITHUB-USERNAME/aggregate
 
-1. Download and install [IntelliJ](https://www.jetbrains.com/idea/) 
+## Setting up the database
 
-1. Fork the Aggregate project ([why and how to fork](https://help.github.com/articles/fork-a-repo/))
+Aggregate supports a variety of DBs, but we strongly recommend you use PostgreSQL first to ensure everything is working. If you wish to use another DB (e.g., Google App Engine, MySQL, or SQLServer databases) after that see [database configurations][docs/database_configurations].
```

Comment:
There is a syntax error in the link to the database configurations doc. I think it should be changed to `[database configurations](docs/database_configurations)` (notice the parenthesis instead of the brackets)